docs/README.md

```diff
@@ -1,177 +1,69 @@
-# VTEX Minicart
+📢 Use this project, [contribute](https://github.com/vtex-apps/breadcrumb) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
+# Minicart
 
-## Description
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
```

Comment:
What is this?

Reply:
I think this is the contributors photos

Reply:
Shouldnt they be at the end?

Reply:
This is the all contributors badge:
![Screen Shot 2019-12-19 at 17 06 18](https://user-images.githubusercontent.com/27777263/71205794-eec16a00-2281-11ea-82c9-326ce1456269.png)